examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>Better do add description of each logical block. For this block better to write sth like "Here we download the dataset by the direct link". There shouldn't be comments for each cell, only for the blocks of cells covering a piece of logic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Looks a little bit too much)</p><p>My fail, let's remove it)))</p>